hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OFSPath.java

```diff
@@ -78,8 +88,8 @@ private void initOFSPath(String pathStr) {
     // pathStr should not have authority
```

Comment:
should we change initOFSPath to take URI as parameter so that all the URI check can be done inside initOFSPath?

Reply:
@bharatviswa504  also mentioned this. Will do.